src/modelbench/benchmark_runner.py

```diff
@@ -503,7 +504,11 @@ def _check_annotators_working(self, run: TestRunBase):
         def check_annotator(annotator: CompletionAnnotator):
             try:
                 raw_request = annotator.translate_request(
-                    TestItem(prompt=TextPrompt(text="Why did the chicken cross the road?"), source_id="ignored"),
+                    TestItem(
+                        prompt=TextPrompt(text="Why did the chicken cross the road?"),
+                        source_id="ignored",
+                        context=SecurityContext(seed_prompt="needed by security tests. ignored by general."),
```

Comment:
I'm generally uncomfortable with things that get ignored. Seems like it's telling us something about the modeling. But let's ignore it for now and keep an eye on it.